Makefile

```diff
@@ -1,9 +1,14 @@
 version=$(shell cat VERSION)
 pkgname=minisphere-$(version)
+os=$(shell uname)
 
 ifndef prefix
+ifeq ($(os), Darwin)
+prefix=/usr/local
+else
 prefix=/usr
 endif
```

Comment:
I was going to complain that using `if` uglies up the makefile, but it looks like I already use it so I can't talk. Carry on. 😜

Reply:
Incidentally, should the install be to `/usr/local/` on all platforms? I think that's standard - see https://www.pathname.com/fhs/pub/fhs-2.3.pdf section 4.8.2

Reply:
When I wrote the makefile, I wasn't that familiar with unix/linux standards, _but_ from what I remember, one of two issues (I don't remember specifically which) drove the decision to use `/usr/`:

* Ubuntu made it difficult to install into `/usr/local/` due to some permissions issue, **OR**
* I was leery about putting stuff in `/usr/local/` because that's where the package manager puts things, and I didn't want to interfere with it doing its job (the thinking is, the package manager keeps track of everything it installs, and here I am putting unaccounted-for stuff in there, so something might go wrong?)

Reply:
I'll leave it for now :)